pages/game/game-result.tsx

```diff
@@ -14,11 +14,11 @@ const GameResult = () => {
   // Get query arguments
   const router = useRouter();
   const { result } = router.query;
-  let isDraw: boolean[] = [];
 
   const [useResult, setResult] = useState<GameResultType>();
   const [useResultComponets, setResultsComponents] = useState<ReactNode[]>();
   const [useWinner, setWinner] = useState<Player>();
+  const [isDraw, setIsDraw] = useState<Boolean[]>([]);
```

Comment:
@ebisuG 
Is this okay to use "Boolean" not the lowercase boolean, primitive type? If it does not cause error, it's okay I think.

Reply:
Check this doesn't cause error.